### PR TITLE
 #357 Bump target SDK version to 28

### LINKED ISSUE #357: AndroidManifest target is too low
Android target SDK version is set in the default AndroidManifest to 21, and that triggers a warning when installing the app.

Target has to be set at least to 26 (28 since latest requirements to submit to Play Store)

### PR DESCRIPTION
**Pull request by jperedadnr**
Also Rename namespace from `xmlns:a` to `xmlns:android`

Fixes #357

### CODE REVIEW

**Comment by johanvos** [comment]
Did you check that this work?
I am afraid that as of 28, a library can not have R+X segments, which our generated lib does.

**Comment by jperedadnr** [comment]
I've checked HelloFX on my Pixel X with Android 10. 
Note that this is now mandatory for publishing apps to the store.

**Comment by johanvos** [comment]
Let's wait for the next GraalVM build, to see if the R/X segments are separated.

File: src/main/resources/native/android/AndroidManifest.xml [hunk 1]
```diff
@@ -1,12 +1,12 @@
 <?xml version='1.0'?>
-<manifest xmlns:a='http://schemas.android.com/apk/res/android' package='com.gluonhq.helloandroid' a:versionCode='1' a:versionName='1.0'>
```
Comment (johanvos) [inline src/main/resources/native/android/AndroidManifest.xml]: I don't think renaming a to android adds anything, so can this be reverted?

Comment (jperedadnr) [reply src/main/resources/native/android/AndroidManifest.xml]: sure

Comment (jperedadnr) [reply src/main/resources/native/android/AndroidManifest.xml]: done

**Pull request merged**